psalm.xml

```diff
@@ -3,6 +3,8 @@
     errorLevel="2"
     phpVersion="8.2"
     resolveFromConfigFile="true"
+    findUnusedBaselineEntry="true"
+    findUnusedCode="false"
```

Comment:
Another flag that needs to be set now. Setting it to `true` causes a lot of new errors, which is why I'd like to do this in a later PR (if we want to do it at all).